prod/docker-compose.yaml

```diff
@@ -0,0 +1,20 @@
+---
+version: "3.4"
+services:
+  fhir:
+    restart: unless-stopped
+    healthcheck:
+      test:
+        wget --spider --quiet
+          http://localhost:8080/fhir/metadata
```

Comment:
nice!

Reply:
seems to work well enough for now. When I intentionally misconfigured the DB connection, `dc ps` was correctly showing `unhealthy`